packages/grid_client/tests/modules/kubernetes.test.ts

```diff
@@ -217,13 +217,25 @@ test("TC1231 - Kubernetes: Deploy a Kubernetes Cluster", async () => {
   const workerPlanetaryIp = result.workers[0].planetary;
   const user = "root";
 
-  //Wait for 20 seconds until the master is ready
-  const wait = await setTimeout(20000, "Waiting for K8s to be ready");
-  log(wait);
-
   //SSH to the master
   const masterSSH = await RemoteRun(masterPlanetaryIp, user);
 
+  //Wait until the cluster is ready
+  let reachable = false;
+  for (let i = 0; i < 40; i++) {
+    await masterSSH.execCommand("source /etc/profile && kubectl get nodes").then(async function (result) {
+      const res = result.stdout;
+      if (res.includes(masterName.toLowerCase()) && res.includes(workerName.toLowerCase())) {
+        reachable = true;
+      }
+    });
+    if (reachable) {
+      break;
+    }
+    const wait = await setTimeout(5000, "Waiting for cluster to be ready");
+    log(wait);
+  }
```

Comment:
can we make it a method and reuse it?

Reply:
added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/390/commits/a1068c537c5469c02465d9e7c7ec6034d2154162

Reply:
forgot to add the utils file to the last commit. added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/390/commits/3b5ca9871fe7817734eaee2a46c0da7c1861ab29